distribution/lib/Standard/Table/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -0,0 +1,82 @@
+from Standard.Base import all
+from Standard.Table.Data.Table import Table
+from Standard.Table.Data.Column import Column
+from Standard.Table.Data.Value_Type import Value_Type
+
+from Standard.Table.Data.Filter_Condition.Filter_Condition import all
+
+type Filter_Condition
+    ## Is less than a value (or another column)?
+    Less than:(Column|Any)
+
+    ## Is less than or equal to a value (or another column)?
+    Equal_Or_Less than:(Column|Any)
+
+    ## Is equal to a value (or another column)?
+    Equal to:(Column|Any)
+
+    ## Is greater than or equal to a value (or another column)?
+    Equal_Or_Greater than:(Column|Any)
+
+    ## Is greater than a value (or another column)?
+    Greater than:(Column|Any)
+
+    ## Is not equal to a value (or another column)?
+    Not_Equal to:(Column|Any)
+
+    ## Is between (inclusive) two values (or columns)?'
+    Between lower:(Column|Any) upper:(Column|Any)
+
+    ## Does the value start with a prefix (Text only)?
+    Starts_With prefix:Text
+
+    ## Does the value end with a suffix (Text only)?
+    Ends_With suffix:Text
+
+    ## Does the value contain the needle (Text only)?
+    Contains substring:Text
+
+    ## Is equal to Nothing?
+    Is_Nothing
+
+    ## Is not equal to Nothing?
+    Not_Nothing
+
+    ## Is the value equal to True (Boolean only)?
+    Is_True
+
+    ## Is the value equal to False (Boolean only)?
+    Is_False
+
+## PRIVATE
```

Comment:
Feel like this can be a member method or a static of the type.

Reply:
I was thinking about it but it felt to me like this would make for too tight coupling - this method is specific to the `Table` aspect of `Filter_Condition`, but once we will be doing filtering on Vector etc. it will be irrelevant.
I kept it in the same file but I feel like it maybe should be moved. Especially that the `Filter_Condition` type itself should probably be moved to Base, I just realised.